mm2src/coins/eth.rs

```diff
@@ -451,15 +491,13 @@ pub struct EthCoinImpl {
     contract_supports_watchers: bool,
     web3_instances: AsyncMutex<Vec<Web3Instance>>,
     decimals: u8,
-    gas_station_url: Option<String>,
-    gas_station_decimals: u8,
-    gas_station_policy: GasStationPricePolicy,
```

Comment:
So gas station will not be used anymore, I agree but hope that no body depends on this. c.c. @cipig